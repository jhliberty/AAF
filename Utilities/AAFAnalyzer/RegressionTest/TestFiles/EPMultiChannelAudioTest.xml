<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE aaf-file SYSTEM "EPDerivationChain.dtd">
<aaf-file>
 <top-level>
  <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
   <source-clip length="0" track-type="audio">
    <clip>
     <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
      <source-clip length="0" track-type="audio">
       <mono-audio name="Mono Audio in Composition and Master" sample-rate-denominator="1" sample-rate-numerator="1">
        <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
         <eoc length="0" track-type="audio"/>
        </timeline-mob-slot>
       </mono-audio>
      </source-clip>
     </timeline-mob-slot>
    </clip>
   </source-clip>
  </timeline-mob-slot>
  <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
   <source-clip length="0" track-type="audio">
    <clip>
     <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
      <source-clip length="0" track-type="audio">
       <multi-channel-audio name="Multi-Channel Audio in Composition and Master" sample-rate-denominator="1" sample-rate-numerator="1">
        <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
         <eoc length="0" track-type="audio"/>
        </timeline-mob-slot>
       </multi-channel-audio>
      </source-clip>
     </timeline-mob-slot>
    </clip>
   </source-clip>
  </timeline-mob-slot>
 </top-level>
</aaf-file>
