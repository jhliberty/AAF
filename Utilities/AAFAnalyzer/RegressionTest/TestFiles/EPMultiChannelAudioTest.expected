        Name:   Edit Protocol Multi-Channel Audio Test
        Desc:   Verify that multi-channel audio is used correctly within the AAF file.
        Result: Failed
        Reason: 
        Covered Requirements: None
        Noted Requirements: None
        Passing Requirements: None
        Info Requirements: None
        Warning Requirements: None
        Failing Requirements: REQ_EP_110

            Name:   Multi-Channel Audio Representation in CompositionMobs and MasterMobs
            Desc:   Within a CompositionMob or MasterMob, multi-channel audio shall be represented using multiple tracks with mono sound Data Definition.  Within a CompositionMob or MasterMob, a multi-channel sound Data Definition shall not be used.
            Node:   18 (SourceMob)
            Result: Failed
            Reason: Multi-Channel Audio File Source "Multi-Channel Audio in Composition and Master" is referenced from a Composition or Master Mob.
            Covered Requirements: None
            Noted Requirements: None
            Passing Requirements: None
            Info Requirements: None
            Warning Requirements: None
            Failing Requirements: REQ_EP_110

